### DISCUSSION TITLE
官网上的案例都看不了了 都是灰的

### DISCUSSION THREAD

**Discussion by kankantian** [zouyaoji/vue-cesium#492] [Q&A]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/vc-viewer
案例现在打不开了， 一通报错
![image](https://github.com/zouyaoji/vue-cesium/assets/8832463/33f57d54-2d6a-40c8-925f-6fe81f5ec8c8)
![image](https://github.com/zouyaoji/vue-cesium/assets/8832463/13fb9572-0b55-4fac-9334-2f20fa6e5260)

**Comment by zouyaoji** [comment]
建议你清一下浏览器zouyaoji.top的缓存记录，或者用无痕浏览模式看看能不能出来。

**Reply by kankantian** [reply]
今天总算是出来了
